docs/Getting Started/Getting-Started-From-Scratch.md

```diff
@@ -1,100 +1,1127 @@
+    ```
+    $ vim static-nodes.json
+    .... paste below lines with enode generated in previous step, port 21000;IP 127.0.0.1 and raft port set as 50000
+    [
+      "enode://70399c3d1654c959a02b73acbdd4770109e39573a27a9b52bd391e5f79b91a42d8f2b9e982959402a97d2cbcb5656d778ba8661ec97909abc72e7bb04392ebd8@127.0.0.1:21000?discport=0&raftport=50000"
+    ] 
+    $ cp static-nodes.json new-node-1
+    ```
+8. Initialize new node with below command.
+    ```
+    $ geth --datadir new-node-1 init genesis.json
+    
+    INFO [06-07|15:45:17.508] Maximum peer count                       ETH=25 LES=0 total=25
+    INFO [06-07|15:45:17.516] Allocated cache and file handles         database=/Users/krish/new-node-1/geth/chaindata cache=16 handles=16
+    INFO [06-07|15:45:17.524] Writing custom genesis block 
+    INFO [06-07|15:45:17.524] Persisted trie from memory database      nodes=1 size=152.00B time=75.344µs gcnodes=0 gcsize=0.00B gctime=0s livenodes=1 livesize=0.00B
+    INFO [06-07|15:45:17.525] Successfully wrote genesis state         database=chaindata                              hash=ec0542…9665bf
+    INFO [06-07|15:45:17.525] Allocated cache and file handles         database=/Users/krish/new-node-1/geth/lightchaindata cache=16 handles=16
+    INFO [06-07|15:45:17.527] Writing custom genesis block 
+    INFO [06-07|15:45:17.527] Persisted trie from memory database      nodes=1 size=152.00B time=60.76µs  gcnodes=0 gcsize=0.00B gctime=0s livenodes=1 livesize=0.00B
+    INFO [06-07|15:45:17.527] Successfully wrote genesis state         database=lightchaindata                              hash=ec0542…9665bf
+    ```
+9. Start your node by first creating a script as below and then starting it:
+    ```
+    $ vim startnode1.sh
+    ... paste below commands. It will start it in the background.
+    #!/bin/bash
+    PRIVATE_CONFIG=ignore nohup geth --datadir new-node-1 --nodiscover --verbosity 5 --networkid 31337 --raft --raftport 50000 --rpc --rpcaddr 0.0.0.0 --rpcport 22000 --rpcapi admin,db,eth,debug,miner,net,shh,txpool,personal,web3,quorum,raft --emitcheckpoints --port 21000 >> node.log 2>&1 &
 
-Your node is now operational and you may attach to it with `geth attach new-node-1/geth.ipc`. This configuration starts Quorum without privacy support as could be evidenced in prefix `PRIVATE_CONFIG=ignore`, please see below sections on [how to enable privacy with privacy transaction managers](#adding-privacy-transaction-manager).
+    $ chmod +x startnode1.sh 
+    $ ./startnode1.sh
+    ``` 
+
+    Your node is now operational and you may attach to it with below commands`. This configuration starts Quorum without privacy support as could be evidenced in prefix `PRIVATE_CONFIG=ignore`, please see below sections on [how to enable privacy with privacy transaction managers](#adding-privacy-transaction-manager).
```

Comment:
Lets convert this paragraph: 

This configuration starts Quorum without privacy support as could be evidenced in prefix `PRIVATE_CONFIG=ignore`, please see below sections on [how to enable privacy with privacy transaction managers](#adding-privacy-transaction-manager)

into a note before the "your node is now opera..."

Reply:
This should be done at every point below too. Thanks!